benchmarks/renderImageItem.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
```

Comment:
This benchmark supercedes the makeARGB one, so we should just delete that file as part of this.

Reply:
Done. I have also changed floating point to use float32.